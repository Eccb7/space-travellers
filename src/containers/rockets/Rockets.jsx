import React from 'react';
import './rockets.css';

function Rockets() {
  return <div>Rockets</div>;
}

export default Rockets;
