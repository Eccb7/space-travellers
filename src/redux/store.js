import { configureStore } from '@reduxjs/toolkit';
import { createLogger } from 'redux-logger';
import missionReducer from './missions/missionSlice';

const logger = createLogger();
const store = configureStore({
  reducer: {
    mission: missionReducer,
  },
  middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
});

export default store;
