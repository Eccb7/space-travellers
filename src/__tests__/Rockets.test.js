import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Rockets from '../containers/rockets/Rockets';

test('renders Rockets component correctly', () => {
  const { container } = render(
    <Provider store={store}>
      <Rockets />
    </Provider>,
  );
  expect(container).toMatchSnapshot();
});
