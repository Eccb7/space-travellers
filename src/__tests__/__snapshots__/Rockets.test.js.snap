// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Rockets component correctly 1`] = `
<div>
  <div
    class="rockets-container"
  >
    <p>
      Loading...
    </p>
  </div>
</div>
`;
